src/components/WebexMemberRoster/WebexMemberRoster.stories.js

```diff
@@ -10,7 +10,7 @@ const Template = (args) => <WebexMemberRoster {...args} />;
 
 export const Space = Template.bind({});
 Space.args = {
-  destinationID: 'room1',
+  destinationID: 'room2',
   destinationType: 'room',
 };
```

Comment:
I would suggest adding a new story for this.


`export const DifferentOrganization = Template.bind({});
DifferentOrganization.args = {
  destinationID: 'room2',
  destinationType: 'room',
};`

Reply:
I think for this scenario, there should be an existing story from the Roster PR but it would get updated here because of the changes to member

Reply:
@karinasigartau0798, by this I meant that you should not modify data and/or stories in the roster component. Please revert this changes

Reply:
Done.